src/HeckeMiscFiniteField.jl

```diff
@@ -33,38 +33,49 @@ end
 # fqPolyRepField
 
 function Base.iterate(F::fqPolyRepField)
-    return zero(F), zeros(UInt, degree(F))
+  return zero(F), zeros(UInt, degree(F))
 end
 
 function Base.iterate(F::fqPolyRepField, st::Vector{UInt})
-    done = true
-    for j in 1:length(st)
-        if st[j] != F.n - 1
-            done = false
-        end
-    end
-
-    if done
-        return nothing
+  done = true
+  for j in 1:length(st)
+    if st[j] != F.n - 1
+      done = false
     end
+  end
 
-    st[1] = st[1] + 1
-    for j in 1:(length(st)-1)
-        if st[j] == F.n
-            st[j] = 0
-            st[j+1] = st[j+1] + 1
-        end
-    end
+  if done
+    return nothing
+  end
 
-    d = F()
-    ccall((:fq_nmod_init2, libflint), Nothing,
-        (Ref{fqPolyRepFieldElem}, Ref{fqPolyRepField}), d, F)
-    for j in 1:length(st)
-        ccall((:nmod_poly_set_coeff_ui, libflint), Nothing,
-            (Ref{fqPolyRepFieldElem}, Int, UInt), d, j - 1, st[j])
+  st[1] = st[1] + 1
+  for j in 1:(length(st) - 1)
+    if st[j] == F.n
+      st[j] = 0
+      st[j + 1] = st[j + 1] + 1
     end
-
-    return d, st
+  end
+
+  d = F()
+  ccall(
+    (:fq_nmod_init2, libflint),
+    Nothing,
+    (Ref{fqPolyRepFieldElem}, Ref{fqPolyRepField}),
+    d,
+    F,
+  )
+  for j in 1:length(st)
+    ccall(
+      (:nmod_poly_set_coeff_ui, libflint),
+      Nothing,
+      (Ref{fqPolyRepFieldElem}, Int, UInt),
+      d,
+      j - 1,
+      st[j],
+    )
```

Comment:
Can't say I'm a huge fan of these additional line breaks in the `ccall`s.

Reply:
I agree. Does it help to increase the allowed line length to something absurd like 200? I don't think there is any other option to keep these as one-liner, is there?

Reply:
I adjusted the rules a bit. To me it looks better now and at least it is not creating ~30000 new lines anymore.